### ISSUE TITLE
sql-drizzle is completely undocumented

### ISSUE THREAD

**Issue by tmcw** [Effect-TS/effect#3796]
### What is the type of issue?

Documentation is missing

### What is the issue?

Effect apparently has a drizzle integration, but it's completely undocumented, no docs on the website,

![CleanShot 2024-10-17 at 11 05 18@2x](https://github.com/user-attachments/assets/fd220568-daff-4e4c-9d9b-19f765c8389d)

And no docstrings in the JSDoc:

![CleanShot 2024-10-17 at 11 12 00@2x](https://github.com/user-attachments/assets/f36f759c-b9e0-4293-9016-2af2c835d432)

**Comment by tmcw** [comment]
This goes for most of the effect addons, afaict - a few are lucky enough to have README.md files, but most don't have any JSDoc descriptions, no examples, no documentation on the website. I think that, more than any other factor, the lack of documentation is the blocker for onboarding my team onto Effect, between the missing JSDoc in the core module #3695 and extensions, there just seems to be a lot of energy toward adding features to the Effect ecosystem and no energy for documenting them.

**Comment by jessekelly881** [comment]
For the moment there's: https://github.com/Effect-TS/effect/blob/main/packages/sql-drizzle/examples/sqlite.ts

**Comment by MindfulLearner** [comment]
Hi @tmcw, I’ve read [[this issue #3695](https://github.com/Effect-TS/effect/issues/3695)](https://github.com/Effect-TS/effect/issues/3695) pointing me here. It would be great if we could start working on improving the documentation of the JSDocs.

Let me know if needs help! I'm trying to learn it so would be a perfect thing to do!
Also not only for me but for everyone else!

**Comment by fubhy** [comment]
Closing this as stale since there is no current confirmation that this documentation gap still needs tracking. If sql-drizzle docs are still missing in the current version, please open a new issue with the current docs/API locations that need coverage.

**Comment by tmcw** [comment]
There are still no docs for drizzle or SQL on the effect website:

<img width="1758" height="1516" alt="Image" src="https://github.com/user-attachments/assets/39fce8cb-89bf-4ef1-a091-b16e489d5158" />

**Comment by fubhy** [comment]
> There are still no docs for drizzle or SQL on the effect website:

Yes, we are going to merge our upcoming v4 release codebase into this repository shortly and are atm. grooming our issue backlog to eliminate stale issues as much as possible. There'll be some collateral damage in this cleanup. Apologies. As we head into RC of Effect v4, we'll shift our focus to documentation, tooling, etc. and also strive to have broader documentation coverage this time.
